hack/generator/pkg/codegen/storage/type_converter.go

```diff
@@ -149,3 +220,8 @@ func (_ *TypeConverter) descriptionForStorageVariant(definition astmodel.TypeDef
 
 	return result
 }
+
+type typeConverterContext struct {
+	isSpec   bool
+	isStatus bool
```

Comment:
`isStatus` doesn't appear to be used anywhere - maybe typeConverterContext could just be a bool or enum value?

Reply:
No longer need this struct at all.